content/en/tutorials/simulating-outages/index.md

```diff
@@ -0,0 +1,224 @@
+The Chaos Plugin is powerful enough to refine outages to particular operations like `PutItem` or `GetItem`, but the objective here is to simulate a failure of entire database service.
+The following configuration will cause all API calls to fail with a 100% failure rate, each resulting in an HTTP 500 status code and a `DatacentreNotFound` error.
+
+{{<command>}}
+curl --location --request PATCH 'http://localhost.localstack.cloud:4566/_localstack/chaos/faults' \
+--header 'Content-Type: application/json' \
+--data '
+[
+    {
+        "service": "dynamodb",
+        "probability": 1.0,
+        "error": {
+            "statusCode": 500,
+            "code": "DatacentreNotFound"
+        }
+    }
+]'
+{{</ command >}}
+
+This makes the database become inaccessible.
+No external client or a LocalStack service can retrieve or add new products, resulting in the API Gateway returning an Internal Server Error.
+
+Downtime and data loss are critical issues to avoid in enterprise applications.
+Fortunately, encountering this issue early in the development phase allows developers to implement effective error handling and develop mechanisms to prevent data loss during a database outage.
+
+### Designing a more resilient system
+
+{{< figure src="arch-2.png" width="800">}}
+
+A possible solution involves setting up an SNS topic, an SQS queue, and a Lambda function.
+The Lambda function will be responsible for retrieving queued items and attempting to re-execute the `PutItem` operation on the database.
+If DynamoDB remains unavailable, the item will be placed back in the queue for a later retry.
+
+{{< command >}}
+$ curl --location 'http://12345.execute-api.localhost.localstack.cloud:4566/dev/productApi' \
+--header 'Content-Type: application/json' \
+--data '{
+    "id": "prod-1003",
+    "name": "Super Widget",
+    "price": "29.99",
+    "description": "A versatile widget that can be used for a variety of purposes. Durable, reliable, and affordable."
+}'
+<disable-copy>
+A DynamoDB error occurred. Message sent to queue.
+</disable-copy>
+{{< /command >}}
+
+If we review the logs, it will show that the `DynamoDbException` has been managed effectively.
+
+```text
+2023-11-06T22:21:40.789 DEBUG --- [   asgi_gw_2] l.services.fis.handler     : FIS handler called with configs: {'dynamodb': {None: [(100, 'DynamoDbException', '500')]}}
+2023-11-06T22:21:40.789  INFO --- [   asgi_gw_2] localstack.request.aws     : AWS dynamodb.PutItem => 500 (DynamoDbException)
+2023-11-06T22:21:40.834 DEBUG --- [   asgi_gw_4] l.services.sns.publisher   : Topic 'arn:aws:sns:us-east-1:000000000000:ProductEventsTopic' publishing '5520d37a-fc21-4a73-b1bf-f9b9afce5908' to subscribed
+'arn:aws:sqs:us-east-1:000000000000:ProductEventsQueue' with protocol 'sqs' (subscription 'arn:aws:sns:us-east-1:000000000000:ProductEventsTopic:0a4abf8c-744a-404a-9ff9-f132e25d1b30')
+```
+
+This element will remain in the queue until the outage is resolved.
+
+### Ending the outage
```

Comment:
do we already have tooling to facilitate this in integration tests? could be nice section here, taking it from just doing and checking it manually to integrate it into a test

Reply:
Not at this time but it is planned in a future workstream